### PR TITLE
RDPP-5722: Fix dependency vulnerabilities

### PR DESCRIPTION
**Pull request by alasdairhurst**

### CODE REVIEW

File: test/logger.js [hunk 1]
```diff
@@ -478,7 +477,7 @@ describe('ADI logging', function () {
 					next();
 				});
 				app.get('/echo', function (req, resp, next) {
-					resp.send({ hello:'world' });
+					resp.send({ hello: 'world' });
```
Comment (alasdairhurst) [inline test/logger.js]: The linter was failing the build so these got auto-fixed

File: test/logger.js [hunk 2]
```diff
@@ -41,10 +41,10 @@ describe('logger', function () {
 		});
 	});
 
-	after(function (done) {
+	after(function () {
 		this.timeout(10000);
 		_console.stop();
-		_util.cleanupTempDirs(done);
```
Comment (alasdairhurst) [inline test/logger.js]: was causing tests to hang (windows)

File: lib/logger.js [hunk 3]
```diff
@@ -447,7 +447,7 @@ function createExpressLogger(app, options) {
 function createDefaultLogger(options) {
 	const ConsoleLogger = require('./console'),
 		consoleLogger = new ConsoleLogger(options),
-		config = _.merge({
+		config = _.mergeWith({
```
Comment (alasdairhurst) [inline lib/logger.js]: merge replacement in v4 of lodash with replacer

**Comment by wickedest** [comment]
The branch is wrong for this issue, "RDPP-5722" should be "rdpp-5727".

**Comment by alasdairhurst** [comment]
@wickedest  typo but don;t think it matters too much for this one-off since it's gonna get deleted pretty quickly anyway :P

**Pull request merged**